components/BottomAppBar/examples/BottomAppBarTypicalUseExample.m

```diff
@@ -0,0 +1,102 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
2017. Check them all.

Reply:
There's been some back and forth debate on this. The other components, including MDCTextField have 2016 in the copyright. Should I still change BottomAppBar to 2017?

Reply:
New code should include the current year.

Reply:
There are some classes in text field that were mistakenly marked as 2016. Once that mistake has been published, you aren't allowed (here) to correct it.